metals/src/main/scala/scala/meta/internal/metals/PackageProvider.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.internal.metals
+
+import org.eclipse.lsp4j.Position
+import org.eclipse.lsp4j.Range
+import org.eclipse.lsp4j.TextEdit
+import org.eclipse.lsp4j.WorkspaceEdit
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+class PackageProvider(private val buildTargets: BuildTargets) {
+
+  def addPackageWorkspaceEdit(path: AbsolutePath): Option[WorkspaceEdit] = {
```

Comment:
`addPackageWorkspaceEdit` -> just `workspaceEdit` should be enough

Reply:
I've chosen that name considering future uses of `PackageProvider`. For example `WorkspaceEdit` for renaming package after file move is going to be a little bit different (I think 2x `\n` shouldn't be added in that case). Anyway, for the current usage it's not needed so I'll change the name.